incubator/vault/values.yaml

```diff
@@ -0,0 +1,62 @@
+# Default values for vault.
+# This is a YAML-formatted file.
+# Declare variables to be passed into your templates.
+replicaCount: 1
+image:
+  repository: vault
+  tag: 0.8.2
+  pullPolicy: IfNotPresent
+service:
+  name: vault
+  type: ClusterIP
+  port: 8200
+ingress:
+  enabled: false
+  # Used to create Ingress record (should used with service.type: ClusterIP).
+  #hosts:
+  #  - chart-example.local
+  #annotations:
+    # kubernetes.io/ingress.class: nginx
+    # kubernetes.io/tls-acme: "true"
+  #tls:
+    # Secrets must be manually created in the namespace.
+    # - secretName: chart-example-tls
+    #   hosts:
+    #     - chart-example.local
+vault:
+  # Only used to enable dev mode. When in dev mode, the rest of this config
+  # section below is not used to configure Vault. See
+  # https://www.vaultproject.io/intro/getting-started/dev-server.html for more
+  # information.
+  dev: true
+  config:
+    # A YAML representation of a final vault config.json file.
+    # See https://www.vaultproject.io/docs/configuration/ for more information.
+    listener:
+      tcp:
+        address: '[::]:8200'
+        tls_disable: 1
+    # See https://www.vaultproject.io/docs/configuration/storage/ for storage backends
+    storage:
+      #consul:
+      #  address: ""
+      #  path: ""
+      #
+      #etcd:
+      #  address: ""
+      #  path: "vault/"
+      #
+      #s3:
+      #  bucket: ""
+      #  region: ""
+      #  access_key: ""
+      #  secret_key: ""
+      #  endpoint: "" # When not using AWS S3
+resources:
```

Comment:
Don't specify defaults for resources.

Reply:
Hi @unguiculus. Why so? We have a discussion on default resources for traefik in https://github.com/kubernetes/charts/issues/2098, knowing your view would be useful.

Reply:
In general, we recommend to leave this a conscious choice for the user. Users should be able to test a chart out of the box on a system with little resources such as Minikube. If you think you have to specify them (and they are really low anyways in this case), then leave it as it is.